src/components/panels/AlertPanel.vue

```diff
@@ -5,8 +5,8 @@
       <b-list-group-item :style="styleObject" v-for="(alert, index) in alerts" :key="alert.id"
       v-bind:class="{ 'list-group-item-borderless': index !== alerts.length - 1, 'list-group-item-borderbottom': index === alerts.length - 1 }" v-if="alert.active">
         <span class="pull-left">
-          <HubImg :src="archwing" name="Archwing Required for Mission" style="li-misssion-decorator" v-if="alert.archwingRequired" />
-          <HubImg :src="nightmare" name="Nightmare Mission" style="li-misssion-decorator" v-if="alert.nightmare" />
+          <HubImg :src="archwing" name="Archwing Required for Mission" style="li-misssion-decorator" v-if="alert.mission.archwingRequired" />
```

Comment:
We should remember to fix the "misssion" typo at some point.

Reply:
i'll fix it now